File: c/code_wars/keep_up_the_hoop.c
#include <stdint.h>

/**
 * This function determines the feedback for a hoop jumping exercise based on
 * the number of successful jumps.
 *
 * @param n The number of successful hoop jumps made by the user.
 *
 * @return A string message encouraging the user to either move on to tricks if
 * they've made 10 or more jumps, or to keep practicing if they've made less
 * than 10 jumps.
 */
char *hoop_count(uint32_t n) {
    return n >= 10 ? "Great, now move on to tricks"
                   : "Keep at it until you get it";
}

File: scala/code_wars/keep_up_the_hoop.scala
/** This function determines the feedback message for a hoop exercise based on the input number.
  *
  * @param n
  *   The input number representing the hoop count. It should be an integer.
  * @return
  *   A string message based on the hoop count. If the count is 10 or more, the function returns "Great, now move on to
  *   tricks". If the count is less than 10, the function returns "Keep at it until you get it".
  */
def hoopCount(n: Int): String = {
  if (n >= 10) {
    "Great, now move on to tricks"
  } else {
    "Keep at it until you get it"
  }
}
